package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "britecharts",
-  "version": "2.4.7",
+  "version": "2.4.8",
```

Comment:
You don't need to update the version here (or in bower.json) since our release script will do it automatically when it's run.

Reply:
Got it!